### DISCUSSION TITLE
quarto render . --to pdf .... how to see progress?

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#2568] [Q&A]
I am rendering a book project with several chapters and parts. After print some metadata, the process stops and seems to be stalled (or running?).  How can I check if its is in progress?  Is there some flag to get a verbose rendering?

command:
```
❯ quarto render . --to pdf
[ 1/11] index.qmd
[ 2/11] intro.qmd
[ 3/11] ai.qmd
[ 4/11] prob.qmd
[ 5/11] mlt.qmd
[ 6/11] info.qmd
[ 7/11] intermezzo.qmd
[ 8/11] ib-method.qmd
[ 9/11] ib-rl.qmd
[10/11] ib-dl.qmd
[11/11] conclusion.qmd

pandoc 
  to: latex
  output-file: index.tex
  standalone: true
  toc: true
  number-sections: true
  top-level-division: chapter
  pdf-engine: xelatex
  variables:
    graphics: true
    tables: true
  default-image-extension: pdf
  
metadata
  crossref:
    chapters: true
  documentclass: kaobook
  papersize: letter
  block-headings: true
  classoption:
    - a4paper
    - fontsize=10pt
    - twoside=false
    - numbers=noenddot
  colorlinks: true
  title: Dissertation
  author: Fred Guth
  date: 9/21/2022
  ````

**Comment by fredguth** [comment]
I guess there was an error somewhere, because it really stoped.

**Reply by dragonstyle** [reply]
Hmm... I have never seen Quarto halt without an error message of some kind. Do you have a document that consistently causes this (if so, I'd be very interested to try it locally so I can try to understand the problem)?

**Reply by fredguth** [reply]
I think I now understand the problem.  Here is the yaml that is generating the problem:
````
...
chapters:
    - index.qmd
    - intro.qmd
    - part: background.qmd
      chapters:
        - ai.qmd
        - prob.qmd
        - mlt.qmd
        - info.qmd
    - part: inter.qmd
      chapters:
        - intermezzo.qmd
    - part: emergence.qmd
      chapters:
        - ib-method.qmd
        - ib-rl.qmd
        - ib-dl.qmd
    - conclusion.qmd
    -...
````

The file inter.qmd didn't exist.  But instead of showing an error, it just stalled.

**Reply by dragonstyle** [reply]
That is interesting - I just tried a handful of different versions of the above but wasn't able to reproduce a hang (for me, when `inter.qmd` doesn't exist, it is just ignored in the render list and treated like text in the output (the text `inter.qmd` literally appears as the part text). 

it looks to me more like we're hanging while in Pandoc (probably because of some filters that we're running). That is likely to be dependent upon content specifically in some way - if you see this again and reproduce it, I would love to be able to take a closer look (e.g. clone a repo or a set of folders that would let me debug it locally).
